doc/over_react_redux_documentation.md

```diff
@@ -0,0 +1,297 @@
+          )(
+            Dom.div()(
+              Dom.h3()('ConnectedBigCounter Store2'),
+              ConnectedBigCounter()(
+                Dom.h4()('ConnectedCounter Store1'),
+                ConnectedCounter()(),
+              ),
+            ),
+          ),
+        ),
+      ), querySelector('#content')
+    );
+```
+
+In the case that you need to have multiple stores, here are the steps to do so:
+1. Create a `Context` instance to provide to the `ReduxProvider` and the components that will be in that context.
+    ```dart
+    final fooContext = createContext();
+    ```
+1. In the `connect` function wrapping the component, pass in the context instance.
+    ```dart
+       UiFactory<BarProps> ConnectedBar = connect<BarState, BarProps>(
+          // ... mapStateToProps
+          context: fooContext,
+       )(Bar);
+    ```
+1. Add an additional `ReduxProvider`, with its `context` prop set to the next Context instance and the `store` prop 
+set to your additional store.
+    ```dart
+       // ... Wrapped in a reactDom.render()
+         (ReduxProvider()..store = store1)(
+           (ReduxProvider()
+             ..store = store2
+             ..context = bigCounterContext
+           )(
+             // ... connected componentry
+           ),
+         )
+    ```
+
+## Using Redux DevTools
+Redux DevTools can be set up easily by adding only a few lines of code.
+> These instructions and additional information can be found [here](https://github.com/MichaelMarner/dart-redux-remote-devtools#middleware-configuration)
+
+1. Add `redux_remote_devtools` and `redux_dev_tools` as dependencies in your `pubspec.yaml`.
+    ```
+    dependencies:
+      redux_dev_tools: 0.4.0
+      redux_remote_devtools: ^0.0.11
+    ```
+    > __NOTE:__ The dependency on `redux_remote_devtools` being `>=0.0.11` is a hard line, as prior to this version 
+    the package was tailored to Flutter and reliant on `dart:io`.
+1. Import `redux_remote_devtools` and `redux_dev_tools` into your store file.
+    ```
+    import 'package:redux_remote_devtools/redux_remote_devtools.dart';
+    import 'package:redux_dev_tools/redux_dev_tools.dart';
+    ```
+1. Change your store to a `DevToolsStore` instance and add the `remoteDevtools` middleware.
+    ```
+    var store = new DevToolsStore<AppState>(/*ReducerName*/, initialState: /*Default App State Object*/, middleware: [remoteDevtools]);
+    ```
+    > __NOTE:__ This should be reverted back to a normal store prior to making your code public (via publishing a 
+        package or deploying to production).
+1. Expose an `initDevtools()` function. This will be used to set up the devtools server.
+    ```
+    var remoteDevtools = RemoteDevToolsMiddleware('127.0.0.1:8000');
+    
+    Future initDevtools() async {
+      remoteDevtools.store = store;
+      return remoteDevtools.connect();
+    }
+    ```
+1. Near the top of your main.dart file call your `initDevtools`.
+    ```dart
+     initDevtools();
```

Comment:
Not necessarily. We found that await could cause confusing behavior because if the consumer does not open the devtools, the the page will never load. Emitting the `await` there allows them to still use to page without having to open up the tools.